<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-BGEVGFM" name="Validating Title" tests="2" failures="0" timestamp="14 Oct 2021 14:33:41 GMT" time="0.083" errors="0">
  <testcase name="@BeforeTest start" time="0.0" classname="Academy.ValidateTitle">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at resources.Base.initializeDriver(Base.java:46)
	at Academy.ValidateTitle.start(ValidateTitle.java:29)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest start -->
  <testcase name="@AfterTest tearDown" time="0.0" classname="Academy.ValidateTitle">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="validateAppTitle" time="0.015" classname="Academy.ValidateTitle">
    <skipped/>
  </testcase> <!-- validateAppTitle -->
  <testcase name="validateHeader" time="0.006" classname="Academy.ValidateTitle">
    <skipped/>
  </testcase> <!-- validateHeader -->
</testsuite> <!-- Validating Title -->
